sale_blanket_order/views/sale_blanket_order_views.xml

```diff
@@ -0,0 +1,370 @@
+                            <field name="name" readonly="1" />
+                        </h1>
+                    </div>
+                    <group>
+                        <group name="group_left">
+                            <field
+                                name="partner_id"
+                                context="{'res_partner_search_mode': 'customer', 'show_address': 1}"
+                                required="state == 'draft'"
+                                options="{&quot;always_reload&quot;: True}"
+                            />
+                            <field name="payment_term_id" />
+                        </group>
+                        <group name="group_right">
+                            <field
+                                name="currency_id"
+                                groups="base.group_multi_currency"
+                                options="{'no_create': True}"
+                            />
+                            <field name="company_id" invisible="1" />
+                            <field name="validity_date" required="state == 'draft'" />
+                        </group>
+                    </group>
+                    <notebook>
+                        <page string="Order Lines" name="order_lines">
+                            <field
+                                name="line_ids"
+                                readonly="state in ['open', 'done', 'expired']"
+                                widget="section_and_note_one2many"
+                            >
+                                <tree editable="bottom">
+                                    <control>
+                                        <create
+                                            name="add_product_control"
+                                            string="Add a product"
+                                        />
+                                        <create
+                                            name="add_section_control"
+                                            string="Add a section"
+                                            context="{'default_display_type': 'line_section'}"
+                                        />
+                                        <create
+                                            name="add_note_control"
+                                            string="Add a note"
+                                            context="{'default_display_type': 'line_note'}"
+                                        />
+                                    </control>
+                                    <field name="display_type" invisible="1" />
+                                    <field name="sequence" widget="handle" />
+                                    <field
+                                        name="product_id"
+                                        required="not display_type"
+                                        column_invisible="display_type"
```

Comment:
please check all `column_invisible="display_type"` occurrences, this is bugging the blanket order form view and also seems functionally incorrect. We shouldn't be able to hide a column for all lines based on the section type of 1 line.

`Caused by: EvalError: Can not evaluate python expression: (bool(display_type))
    Error: Name 'display_type' is not defined
    EvalError: Can not evaluate python expression: (bool(display_type))
    Error: Name 'display_type' is not defined
        at evaluateExpr 
        at evaluateBooleanExpr
        at Object.evalViewModifier 
        at SectionAndNoteListRenderer.evalColumnInvisible 
        at http://localhost:8069/web/assets/8cb151b/web.assets_web.min.js:9349:9
        at Array.filter (<anonymous>)
        at SectionAndNoteListRenderer.getActiveColumns 
        at SectionAndNoteListRenderer.setup`